fe2-android/app/src/main/java/com/github/dedis/student20_pop/model/package-info.java

```diff
@@ -1,6 +1,4 @@
 /**
- * Classes used to model the different entities
- *
- * <p>The classes inside this package contain the basic models.
+ * The package model represents the different objects of the application.
```

Comment:
`business objects`, `entities`, or something akin?

objects could be a very overloaded term